src/probnum/filtsmooth/statespace/sde.py

```diff
@@ -164,7 +164,8 @@ def forcevec(self):
         return self._forcevec
 
     @property
-    def dispersionmatrix(self):  # pylint-disable: invalid-overridden-method
+    def dispersionmatrix(self):
+        # pylint: disable=invalid-overridden-method
```

Comment:
Why is this needed? I suspect it's because `dispersionmatrix` is not a property in SDE, but it is here, right? Is this intended?

Reply:
yes, this is an awkward spot in between a function that returns the dispersion at time $t$, though for the rest of the LTI SDE matrices they are properties.